docs/vscode-example/tasks.json

```diff
@@ -0,0 +1,58 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build debug",
+            "detail": "Build in debug mode",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "make",
+                "build"
+            ]
+        },
+        {
+            "label": "build release",
+            "detail": "Build in release mode",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "make",
+                "build-release"
+            ]
+        },
+        {
+            "label": "clean",
+            "detail": "Clean the target build directory",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "make",
+                "clean"
+            ]
+        },
+        {
+            "label": "rust format",
+            "detail": "Auto format the code",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "fmt"
+            ]
+        },
+        {
+            "label": "run the tests",
+            "detail": "Run the tests",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "test"
+            ]
+        },
```

Comment:
i'll also change this one to use the cargo make test which knows to run multi phase tests (tests for single thread + multi thread + custom tasks for tests)